spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/task/TaskExecutorProperties.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2012-2016 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.boot.autoconfigure.task;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+/**
+ * Configuration properties for the task executor.
+ *
+ * @author Vedran Pavic
+ * @since 1.4.0
+ */
+@ConfigurationProperties(prefix = "spring.task")
+public class TaskExecutorProperties {
+
+	/**
+	 * Maximum pool size for the executor.
+	 */
+	private Integer maxPoolSize = 10;
```

Comment:
I could see a `spring.task.pool.max-size` in there to avoid multiple related properties in the root namespace.